Running-Mastodon/Serving_a_different_domain.md

```diff
@@ -74,6 +74,14 @@ location = /.well-known/host-meta {
 
 in example.org's server block.
 
+Alternatively, you can serve the file from your hosting. The full content of the file:
+```XML
+<?xml version="1.0" encoding="UTF-8"?>
+<XRD xmlns="http://docs.oasis-open.org/ns/xri/xrd-1.0">
+  <Link rel="lrdd" type="application/xrd+xml" template="https://somedomain.org/.well-known/webfinger?resource={uri}"/>
```

Comment:
this needs to be `https://social.example.org` to match the rest of the documentation.

Reply:
You're right, my bad.